oap-server/analyzer/agent-analyzer/src/main/java/org/apache/skywalking/oap/server/analyzer/provider/AnalyzerModuleConfig.java

```diff
@@ -60,6 +62,23 @@ public class AnalyzerModuleConfig extends ModuleConfig {
     @Setter
     @Getter
     private DBLatencyThresholdsAndWatcher dbLatencyThresholdsAndWatcher;
+
+    @Setter
+    @Getter
+    private String slowCacheWriteThreshold = "default:20";
+
+    @Setter
+    @Getter
+    private CacheWriteLatencyThresholdsAndWatcher cacheWriteLatencyThresholdsAndWatcher;
+
+    @Setter
+    @Getter
+    private String slowCacheReadThreshold = "default:20";
+
+    @Setter
+    @Getter
+    private CacheReadLatencyThresholdsAndWatcher cacheReadLatencyThresholdsAndWatcher;
```

Comment:
With all these new configs and dynamic configuration watchers added, the relative docs should be updated too.